Jenkinsfile_parameterized

```diff
@@ -15,6 +15,22 @@ properties([
   pipelineTriggers([[$class: 'GitHubPushTrigger']])
 ])
 
+def secrets = [
```

Comment:
I am not sure why we need this ?

Reply:
I thought we need it as it still exists in the processor. so, can I revert changes in Jenkins files?